NEWS.md

```diff
@@ -1,5 +1,8 @@
 # pkgdown (development version)
 
+* Environment variable ABORT_ON_MISSING_TOPICS can be used to enable/disable
```

Comment:
Can you suggest a slightly more generic name? I think we'll also want to apply this to articles (https://github.com/r-lib/pkgdown/issues/2055). And maybe it should have a `PKGDOWN_` prefix?